helm/export/values.yaml

```diff
@@ -0,0 +1,40 @@
+
+nuxeo:
+  fullnameOverride: export
+  nuxeo:
+    podAnnotations:
+      preview/scm.ref: ${SCM_REF}
+    podLabels:
+      branch: export
```

Comment:
either template with the branch name or remove, not strictly necessary

Reply:
Don't we need to be able to monitor pods usage with these labels/annotations? As explained above, i'm not sure which makes sense for these use cases, so please let me know.

Reply:
(removed these anyway)

Reply:
We're not currently using the labels, they're put to anticipate some kind of monitoring, not yet defined.
`export` is fine as you explained the job is not related to a given branch